docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -127,25 +101,20 @@ Finally, during the data plane adoption, you will have to take care of several a
         fixedIP: 192.168.122.100
         name: ctlplane
         subnetName: subnet1
-```
-
-- expand SSH range for the firewall configuration to include both subnets:
+----
 
-```yaml
+. Expand the SSH range for the firewall configuration to include both subnets:
++
+----
         edpm_sshd_allowed_ranges:
         - 192.168.122.0/24
         - 172.31.0.0/24
-```
-
-This is to allow SSH access from the new subnet to the adopted nodes as well as
-the old one.
-
----
-
-Since you are applying new network configuration to the nodes, consider also
-setting `edpm_network_config_update: true` to enforce the changes.
+----
++
+This provides SSH access from the new subnet to the adopted nodes as well as
+the old nodes.
```

Comment:
Is “old nodes” correct?

Reply:
I think this sentence refers to "old subnets", not "old nodes". (It's communicating that the example above sets up SSH daemon running on data plane nodes to accept connections from two ranges - one that is adopted from the pre-adopted setup (172.x in the example), and one subnet that is newly defined for the new control plane services (192.x).

---

As alluded above in a different comment, we may want to use a different word for "old", though it probably applies for any use of "old" in the documentation, not just in networking / subnet context. These are not really "old" as much as "resources / nodes / subnets that originate from the setup before adoption".

Reply:
Maybe "RHOSP" or "previous" would be a better term.